server/src/application.py

```diff
@@ -54,15 +45,21 @@ def application(environ, start_response):
         # Respond to a GET request.
         if environ['REQUEST_METHOD'] == 'GET':
             # Authenticate.
-            res = ldbdwauth.check_authorization_gridmap(environ, environ['REQUEST_METHOD'], environ['REQUEST_URI'], False)
+            try:
+                res = ldbdsauth.check_authorization_scitoken(environ, environ['REQUEST_METHOD'], environ['REQUEST_URI'], False)
+            except:
```

Comment:
Please don't use empty `except`, I presume this should only protect against `KeyError` which indicates that no token was given

```suggestion
            except KeyError:
```